documentation/expanding.md

```diff
@@ -0,0 +1,196 @@
+            "A": "pe-server-6251cd-0.us-west1-a.c.slice-cody.internal",
+            "B": null
+          },
+          "postgresql": {
+            "A": "pe-psql-6251cd-0.us-west1-a.c.slice-cody.internal",
+            "B": "pe-psql-6251cd-1.us-west1-b.c.slice-cody.internal"
+          },
+          "compilers": {
+            "A": [
+              "pe-compiler-6251cd-0.us-west1-a.c.slice-cody.internal",
+              "pe-compiler-6251cd-1.us-west1-b.c.slice-cody.internal"
+            ],
+            "B": [
+    
+            ]
+          }
+        }
+      }
+    Successful on 1 target: pe-server-6251cd-0.us-west1-a.c.slice-cody.internal
+    Ran on 1 target in 2.56 sec
+
+## Adding Compilers with peadm::add_compiler
+
+Standard deployments are the only architecture of the three which do not have Compilers available upon initial deployment, the lack of them is what differentiates Standard from Large. Architecture has no effect on the process for adding Compilers to a deployment. The [peadm::add_compiler](https://github.com/puppetlabs/puppetlabs-peadm/blob/main/plans/add_compiler.pp) plan functions identical in all three architecture, even if you're adding the 1st or the 100th but some options do change slightly depending.
+
+### Adding Compilers to Standard and Large without disaster recovery
+
+The command invocation is identical when adding compilers to a Standard or Large deployment if disaster recovery (DR) is not enabled and a Replica has not been provisioned. Take note of the values for `avail_group_letter` and `primary_postgresql_host`, in this **no DR**  scenario, the value of these parameter will always be set to **A** and the FQDN of the Primary Puppet server.
+
+    bolt plan run peadm::add_compiler primary_host=<primary-server-fqdn> compiler_host=<new-compiler-fqdn> avail_group_letter=A primary_postgresql_host=<primary-server-fqdn>
+
+### Adding Compilers to Extra Large without disaster recovery
+
+When adding a compiler to an Extra Large deployment in a **no DR** scenario, the only difference is that the `primary_postgresql_host` changes to be the value of the Primary PE-PostgreSQL server as opposed to the Primary Puppet server.
+
+    bolt plan run peadm::add_compiler primary_host=<primary-server-fqdn> compiler_host=<new-compiler-fqdn> avail_group_letter=A primary_postgresql_host=<primary-postgresql-server-fqdn>
+
+### Adding Compilers to Standard and Large when disaster recovery has been enabled
+
+When disaster recovery (DR) is enabled and a Replica provisioned, PEADM creates a second availability group. You must take this into consideration when adding new compilers and ensure you are assigning appropriate values for the group the compiler is targeted for. It is a good idea to keep these two availability groups populated with an equal quantity of compilers. Besides the value of `avail_group_letter` being dependent on which group the new compiler is targeted towards, the value of `primary_postgresql_host` will also vary.
```

Comment:
* `availability group` -> `role letter`? Those terms are both used and it's not entirely clear from the docs what it is, exactly

Reply:
It is unfortunate that role letter is used in the `peadm::get_peadm_config` but all the CLI parameters refer to availability group. I added a **Notes** section at the top to highlight these inconsistency issues.